applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -95,9 +98,10 @@ static void connected_cb(struct bt_conn *conn, uint8_t err)
 	LOG_INF("Connected: %s", addr);
 
 #if defined(CONFIG_BT_MAX_CONN)
```

Comment:
Because BT_MAX_CONN is used on the line below. If you compile for BIS then this define is not set and the compiler will complain. We could fix it as we do in bt_volume:
#ifndef CONFIG_BT_MAX_CONN
#define VCS_PEER_NUM 2
#else
#define VCS_PEER_NUM CONFIG_BT_MAX_CONN
#endif

Reply:
I still feel there should be a better way of solving this, but we can let it be for now